definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -1825,5 +1825,1349 @@
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "13/12/2018",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "transferCaseFromCTSCreferLA",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "13/12/2018",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "transferCaseFromCTSCpayment",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "13/12/2018",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "transferCaseFromCTSCpronounce",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "13/12/2018",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "transferCaseFromCTSCreissue",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "13/12/2018",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "transferCaseFromCTSCissued",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "13/12/2018",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "transferCaseFromCTSCpendReject",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "13/12/2018",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "transferCaseFromCTSCrejected",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "13/12/2018",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "transferCaseFromCTSCsubmitted",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "transferCaseToNewRDC",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "updateContactDetails",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "updateDueDate",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "uploadDocument",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "withdrawn",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "AddNote",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "aosReceivedNoAdConStarted",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "aosReceivedNoAdConOverdue",
```

Comment:
Why do we have to repeat these events? I thought any super-user would have the other court-admin roles assigned to them as well, like `courtadmin_beta`?

Reply:
i.e. why repeat permissions?

Reply:
I think the idea is to have them self-sufficent and not relay on compound roles - that's what I gathered from https://tools.hmcts.net/jira/browse/DIV-4579
@gavinclark  can confirm though..

Reply:
Seems un-necessary when they could have the permissions through the existing role? That's kind of the point of having this ability in CCD. It becomes error prone in terms of maintaining permissions across both user role types as well, each time a new event is introduced.

Reply:
Fair enough, hopefully @gavinclark can chip in soon!

Reply:
Updated as per discussion on DIV-4579